site/content/en/docs/Guides/Matchmaker/_index.md

```diff
@@ -46,29 +49,37 @@ Open Match allows for concurrent Match Functions to execute on the same player p
 
 ### External Components
 
-Although Open Match based Matchmaker offers core matchmaking functionality, some other Game Services need to handle key functionality such as handling player connections, fetching DGS (Dedicated Game Server) allocation etc. These services interact with Open Match to enable key matchmaking scenarios. Here is an overview of these services that are external to an Open Match based Matchmaker but are a part of an E2E setup:
+Although Open Match based Matchmaker offers core matchmaking functionality, other Game Services will be needed to handle
```

Comment:
unintended line break here?

Reply:
It was intended :smile: my editor will automatically hard wrap lines at 120 chars across.

I couldn't edit the lines that are as wide as they now, it's too hard to see it on a single screen.

Hope that was okay.